### ISSUE TITLE
Add reminder to update all locales when changing request address in just one.

### ISSUE THREAD

**Issue by FOIMonkey** [mysociety/alaveteli#6947]
Related to #3134 but slightly different. As the majority of authorities on WDTK only have an address for one locale, it is very easy to forget that you sometimes need to change two addresses when updating contact details.

Dw i wedi blino heddiw, so I completely overlooked the note to remember to update all locales when fixing a Welsh authority and only updated the English version. This would have meant that only one language version was up to date had I not remembered later. It would be good if some sort of warning was displayed if you are making a change which means that the request addresses for different locales are different when they used to be the same.

**Comment by RichardTaylor** [comment]
I agree. 

We already have a blue in-line tip text saying:

>Make sure you add the request email for all locales that have translated attributes. 

Where relevant this should become red and say 

>You have locales with translated attributes which lack a request address

or in better, in plain English

>Please provide an email address on each language tab in use

Other ideas to address the underlying issue:

*  show an icon and text to indicate if there is, or is not, content in a locale tab on the body edit page. The most intuitive thing might be the words "in use". So tab titles of eg. English (in use), French (not used). 

* Show content from the locale tabs on the body admin page


Fixing #3134 to allow a fallback to the default locale address would be preferable and would negate the need for the alert. It would still be good to flag to admins that localisation is present for a body though.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
